server/protocol.c

```diff
@@ -682,359 +682,13 @@ static int field_name_len(const char *field)
     return end - field;
 }
 
-static const char m_invalid_str[] = "-";
```

Comment:
All this has moved to `modules/http/`.

Reply:
I'm not sure it needed to, it makes the changes harder to follow and protocol.c seems to be calling ap_h1_ specific functions still.
Maybe move them after the functional changes if needed?